src/metadata/bpm.cpp

```diff
@@ -0,0 +1,60 @@
+#include "metadata/bpm.h"
+
+namespace Mixxx {
```

Comment:
There is already a class named "ReplayGain" in the global namespace in 
lib/replaygain/replaygain.h. We definitely need a separate namespace for 
this Mixxx class with the same name! I propose to enclose a class with the 
very common name "Bpm" into our namespace to avoid any naming conflicts with 
other libraries.

Note: I will move the files of our ReplayGain class from util/ to track/ 
like all other track related properties.

On 12/27/2015 05:53 PM, RJ Ryan wrote:
>
> In src/metadata/bpm.cpp 
> <https://github.com/mixxxdj/mixxx/pull/728#discussion_r48456436>:
>
> > @@ -0,0 +1,60 @@
> > +#include "metadata/bpm.h"
> > +
> > +namespace Mixxx {
>
> I see that you've started putting a namespace around everything. I'd like 
> to revisit whether we need this considering the plan to remove the Source 
> plugin system.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/mixxxdj/mixxx/pull/728/files#r48456436>.
>